prov/efa/src/rdm/efa_rdm_protocol.h

```diff
@@ -233,7 +233,7 @@ struct efa_rdm_readrsp_hdr {
 	uint64_t seg_length;
 };
 
-EFA_RDM_ENSURE_HEADER_SIZE(efa_rdm_ctsdata_hdr, sizeof (struct efa_rdm_ctsdata_hdr));
+EFA_RDM_ENSURE_HEADER_SIZE(efa_rdm_readrsp_hdr, 24);
```

Comment:
Oops. If you want to change it to the previous/intended logic:
```suggestion
EFA_RDM_ENSURE_HEADER_SIZE(efa_rdm_readrsp_hdr, sizeof (struct efa_rdm_ctsdata_hdr));
```

Reply:
fixed